BookingsApi/BookingsApi.DAL/Queries/GetAllCaseTypesQuery.cs

```diff
@@ -8,6 +10,12 @@ namespace BookingsApi.DAL.Queries
 {
     public class GetAllCaseTypesQuery : IQuery
     {
+        public GetAllCaseTypesQuery(bool hideExpired)
```

Comment:
just wondering, could we call this `showExpired` and default it to false? That way it's positive logic and it would filter out the expired case types by default, which I think is what we want going forward

Reply:
Good point. I intentionally didn't put a default value in because I wanted to find the usages and where quickly to help workout where best to set it to true or false